### DISCUSSION TITLE
How to optimise Xeon E5-4610 CPU affinity？

### DISCUSSION THREAD

**Discussion by goliath525** [swar/Swar-Chia-Plot-Manager#827] [Q&A]
Dears,

I have a DELL R830 server which contains 4 Xeon E5-4610 v4 CPUs (totally 40cores and 80threads), how could i config the CPU affinity to optimise the plotting process？

**Comment by daozi7272** [comment]
I have the same problem as you for server 4* e5 4657L v2.can't work with cpu affinity.

**Reply by tracy32792** [reply]
I also have the same problem, double CPU binding can only start part of the task

**Reply by HFBerman87** [reply]
Same problem too.
